### PR TITLE
Volkswagen: Tiguan R-Line 2024 Fingerprint 

### PR DESCRIPTION
**Pull request by hodgiers**
**Volkswagen: Tiguan R-Line 2024 **

**Tested and verified with route: 9102455c32396e54|2023-12-04--15-53-50
https://connect.comma.ai/9102455c32396e54/1701723229935/1701728028999**

### CODE REVIEW

**Comment by sshane** [comment]
Can you bump the MY here? https://github.com/commaai/openpilot/blob/09a9ba6de1603fbcda0f883d5298b3f80f8295e1/selfdrive/car/volkswagen/values.py#L242

**Comment by hodgiers** [comment]
> Can you bump the MY here?
> 
> https://github.com/commaai/openpilot/blob/09a9ba6de1603fbcda0f883d5298b3f80f8295e1/selfdrive/car/volkswagen/values.py#L242

Done

**Comment by sshane** [comment]
Also run ./selfdrive/car/docs.py if you can, if your repo is open to pushes from PRs I can as well.

**Comment by sshane** [comment]
Going to close since I can't push to update docs

File: selfdrive/car/volkswagen/values.py [hunk 1]
```diff
@@ -897,7 +897,6 @@ def init_make(self, CP: car.CarParams):
     ],
     (Ecu.fwdRadar, 0x757, None): [
       b'\xf1\x872Q0907572AA\xf1\x890396',
```
Comment (sshane) [inline selfdrive/car/volkswagen/values.py:899]:
```suggestion
      b'\xf1\x872Q0907572AA\xf1\x890396',
      b'\xf1\x872Q0907572AB\xf1\x890397',
```

File: selfdrive/car/volkswagen/values.py [hunk 2]
```diff
@@ -862,7 +863,6 @@ def init_make(self, CP: car.CarParams):
       b'\xf1\x870DL300014C \xf1\x893703',
       b'\xf1\x870DD300046K \xf1\x892302',
       b'\xf1\x870GC300013P \xf1\x892401',
```
Comment (sshane) [inline selfdrive/car/volkswagen/values.py:865]:
```suggestion
      b'\xf1\x870GC300013P \xf1\x892401',
      b'\xf1\x870GC300046Q \xf1\x892802',
```

File: selfdrive/car/volkswagen/values.py [hunk 3]
```diff
@@ -830,7 +830,6 @@ def init_make(self, CP: car.CarParams):
       b'\xf1\x8704E906027NB\xf1\x899504',
       b'\xf1\x8704L906026EJ\xf1\x893661',
       b'\xf1\x8704L906027G \xf1\x899893',
```
Comment (sshane) [inline selfdrive/car/volkswagen/values.py:832]:
```suggestion
      b'\xf1\x8704L906027G \xf1\x899893',
      b'\xf1\x8705E906018BS\xf1\x890914',
```

File: selfdrive/car/volkswagen/values.py [hunk 4]
```diff
@@ -239,7 +239,7 @@ def init_make(self, CP: car.CarParams):
   CAR.TAOS_MK1: VWCarInfo("Volkswagen Taos 2022-23"),
   CAR.TCROSS_MK1: VWCarInfo("Volkswagen T-Cross 2021", footnotes=[Footnote.VW_MQB_A0]),
   CAR.TIGUAN_MK2: [
-    VWCarInfo("Volkswagen Tiguan 2018-23"),
+    VWCarInfo("Volkswagen Tiguan 2018-24"),
```
Comment (sshane) [inline selfdrive/car/volkswagen/values.py:242]:
```suggestion
    VWCarInfo("Volkswagen Tiguan 2018-23),
```

File: selfdrive/car/volkswagen/values.py [hunk 5]
```diff
@@ -239,7 +239,7 @@ def init_make(self, CP: car.CarParams):
   CAR.TAOS_MK1: VWCarInfo("Volkswagen Taos 2022-23"),
   CAR.TCROSS_MK1: VWCarInfo("Volkswagen T-Cross 2021", footnotes=[Footnote.VW_MQB_A0]),
   CAR.TIGUAN_MK2: [
-    VWCarInfo("Volkswagen Tiguan 2018-23"),
+    VWCarInfo("Volkswagen Tiguan 2018-23),
```
Comment (sshane) [inline selfdrive/car/volkswagen/values.py:242]:
```suggestion
    VWCarInfo("Volkswagen Tiguan 2018-23"),
```

**Pull request merged**